wiki/BBCode/en.md

```diff
@@ -1,581 +1,417 @@
-| [heading]Your text here[/heading]     | <span style="text-shadow: 0px 0px 10px rgb(239, 207, 225); color: rgb(204, 46, 138); margin-bottom: 10px; font-size: 160%;">Your text here</span> |
-
-Stylish title \#2
------------------
-
-<table>
-<thead>
-<tr class="header">
-<th><p>Purpose</p></th>
-<th><p>Use this to insert a big fancy purple title followed by a line. Primarily used for design purposes.</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Dedicated button?</p></td>
-<td><p>No: you have to type this by hand.</p></td>
-</tr>
-<tr class="even">
-<td><p>Note</p></td>
-<td><p><strong>Only works within the <a href="http://osu.ppy.sh/forum/12">Beatmaps</a> forum!</strong><br />
-For a weird reason, it also doesn't work in previews but works after having posted the message.</p></td>
-</tr>
-<tr class="odd">
-<td><p>Syntax</p></td>
-<td><p>Write this</p></td>
-</tr>
-<tr class="even">
-<td><pre><code>[Your text here]</code></pre></td>
-<td><div style="color: rgb(67, 63, 117); margin: 1px 3px 5px 0px; padding-left: 8px; border-bottom: 1px solid rgb(190, 187, 205); box-shadow: 0px 3px 2px -3px rgb(190, 187, 205); font-size: 140%; font-weight: bold;">
-<p>Your text here</p>
-</div></td>
-</tr>
-</tbody>
-</table>
-
-White box
----------
-
-<table>
-<thead>
-<tr class="header">
-<th><p>Purpose</p></th>
-<th><p>Use this to insert a white box in which you can write text. Primarily used for design purposes.</p></th>
-</tr>
-</thead>
-<tbody>
-<tr class="odd">
-<td><p>Dedicated button?</p></td>
-<td><p>No: you have to type this by hand.</p></td>
-</tr>
-<tr class="even">
-<td><p>Syntax</p></td>
-<td><p>Write this</p></td>
-</tr>
-<tr class="odd">
-<td><pre><code>[notice]A first line
-A second line[/notice]</code></pre></td>
-<td><div style="background: none repeat scroll 0% 0% rgb(249, 247, 254); border: 1px solid rgb(225, 223, 231); margin: 6px; padding: 5px;">
-<p>A first line<br />
-A second line</p>
-</div></td>
-</tr>
-</tbody>
-</table>
+# BBCode
+
+![The edit box in the forums](img/fourms.jpg "The edit box in the forums")
+
+**BBCode** is a markup syntax that is used in the osu!forums and, to a larger extent, the vast majority of forums on the internet to enable rich text formatting.
+It is made up of tags that surround the text to enrich and, sometimes, attributes.
+
+In the osu!website, BBCode is used in forum posts, signatures and userpages.
+This article is a listing of the available tags.
+
+## Notes
+
+Know that the osu!fourms gives you a few format buttons that you could use; however, you could just type them in yourself.
+
+BBCode tags are case insensitive.
+
+### BBCode Button Behavior
+
+If you highlight the text that you want to format then click on one of them, e.g. bold, italics, strikethrough, or underline, it will add said format code to the highlighted text automatically.
+
+If you click the button without highlighting text, it will surround the tags around your text cursor.
```

Comment:
I feel like this discludes the reader from what they are supposed to do/could do.. but if you insist